pkg/admission/controller_test.go

```diff
@@ -0,0 +1,13 @@
+package admission
+
+import (
+	"testing"
+
+	"k8s.io/api/admissionregistration/v1beta1"
+)
+
+func TestCertProvisionerInit(t *testing.T) {
+	p := &CertProvisioner{}
```

Comment:
You need to use a fake client like `Client: fake.NewFakeClient()`.
Otherwise, it will try find the kubeconfig which doesn't exist in the test env.